src/mcp/CONTRIBUTING.md

```diff
@@ -0,0 +1,186 @@
+
+## Audience
+
+If you are a developer interested in contributing to the MCP server, this is the
+documentation for you! This guide describes how to be successful in contributing
+to our repository.
+
+## Getting Started
+
+The Firebase MCP server lives alongside the Firebase CLI in the [firebase/firebase-tools][gh-repo] repo.
+It lives here so that it can share code for authentication, API calls, and utilities with the CLI.
+
+External developers: If you're interested in contributing code, get started by
+[making a fork of the repository for your GitHub account](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Internal developers: Go to go/firebase-github-request and ask for access to this repo. You should work
+off of sepearate branches in this repo, to ensure that all CI runs correctly for you.
+
+### Contribution Process
+
+External developers: The preferred means of contribution to the MCP server is by creating a branch in your
+own fork, pushing your changes there, and submitting a Pull Request to the
+`master` branch of `firebase/firebase-tools`.
+
+Internal developers: Instead of working off of a fork, please make a branch on [firebase/firebase-tools][gh-repo]
+named <yourInitialsOrLDAP>-<feature> (for example, `jh-dataconnect-tools`)
+
+If your change is visible to users, it should be in the
+[changelog](https://github.com/firebase/firebase-tools/releases). Please
+add an entry to the `CHANGELOG.md` file. This log is emptied after every release
+and is used to generate the release notes posted in the
+[Releases](https://github.com/firebase/firebase-tools/releases) page. Markdown
+formatting is respected (using the GitHub style).
+
+NOTE: Any new files added to the repository **must** be written in TypeScript
+and **must** include unit tests. There are very few exceptions to this rule.
+
+After your Pull Request passes the tests and is approved by a Firebase CLI team
+member, they will merge your PR. Thank you for your contribution!
+
+### Setting up your development environment
+
+Please follow the instructions in the [CLI's CONTRIBUTING.md](https://github.com/firebase/firebase-tools/blob/master/CONTRIBUTING.md#setting-up-your-development-environment) to get your development environment set up.
+
+There are a few extra things to set up when developing the MCP server.
+
+### Testing with the MCP Inspector
+
+During early development, you will want to test that your tools outputs what you expect, without burning tokens.
+The easiest way to do this is the [MCP inspector](https://github.com/modelcontextprotocol/inspector). From a
+Firebase project directory, run:
+
+```
+npx -y @modelcontextprotocol/inspector
+```
+
+This will print out a localhost link to a simple testing UI. There, you can configure the MCP server
+and manually list and execute tools.
+
+```
+Transport Type
+STDIO
+Command
+firebase
+Arguments
+experimental:mcp
+
+```
```

Comment:
This block looks a bit weird.

https://screenshot.googleplex.com/8QM9ZjTCAvG7ozG

Shall we maybe include a screenshot of it?